packages/labs/gen-utils/package.json

```diff
@@ -10,9 +10,35 @@
   "main": "index.js",
   "repository": "lit/lit",
   "scripts": {
-    "build": "tsc",
-    "build:watch": "tsc --watch",
-    "test": "uvu test '.*_test\\.js$'"
+    "build": "wireit",
+    "test": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../analyzer:build"
+      ],
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "lib",
+        "test",
+        "index.{js,js.map,d.ts,d.ts.map}"
+      ],
+      "clean": "if-file-deleted"
+    },
+    "test": {
+      "command": "uvu test '.*_test\\.js$'",
+      "dependencies": [
+        "build",
+        "../../lit:build"
```

Comment:
I've been putting dependency `build` inside `build`. So `build` means "everything you need to actually run this thing".

Reply:
That implies factoring out a `build:ts` here.

Reply:
This is only needed for the tests to pass, and it's not a static dependency of the tests, but of the generated files. The build task does not depend on this package contents in any way.

This would be clearer if the tests were split into a generate task and test task, but I wanted to avoid changing code in this PR.